pkgs/top-level/all-packages.nix

```diff
@@ -7886,6 +7886,8 @@ in
 
   rarian = callPackage ../development/libraries/rarian { };
 
+  ratarmount = python3Packages.callPackage ../tools/archivers/ratarmount { };
```

Comment:
According to the readme, this is meant to also be used as an application. Do you mind adding the following to `pkgs/top-level/all-packages.nix`:
```
  <pkg-name> = with python3Packages; toPythonApplication <python-module>;
```
More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)

Reply:
ah, never mind, this was not in `python-modules`. Disregard